src/Apps/Order/Routes/Shipping2/Shipping2.jest.tsx

```diff
@@ -1211,6 +1266,8 @@ describe("Shipping", () => {
               recommendedAddress
             )
 
+            // TODO: Arta quotes don't appear in the updated order relay object.
+            // Do we need to do something to refresh the relay store in testing?
             mockCommitMutation
```

Comment:
Didn't fix address verification tests for saved addresses because it can't find the shipping quotes. Left note here for later but it could be a red herring.

Reply:
Briefly talked with @damassi about this. No definite answer yet.